app/static/src/app/components/mixins/selectVariables.ts

```diff
@@ -0,0 +1,77 @@
+import { Store } from "vuex";
+import { computed } from "vue";
+import { AppState } from "../../store/appState/state";
+import { GraphsAction } from "../../store/graphs/actions";
+
+export interface SelectVariablesMixin {
```

Comment:
very small nitpick.. can this be a compasable instead just to match the vue way

Reply:
What change are you suggesting? It is a composable, just using the old "mixin" terminology since we already have a mixins folder in wodin. Could maybe a separate ticket to make the wodin mixins more composable-ish (rename the folder, the interfaces and prefix with `use..`)..?

Reply:
ohh yeah i remember.. nope dont worry should be sweet